### DISCUSSION TITLE
Old Modena 2.13 to Ultima 1.0 or 2.0

### DISCUSSION THREAD

**Discussion by chuckcMTL** [primefaces/community#1033] [PrimeFaces Templates] [answered]
Hi,

Several years back we purchased Ultima. The intent was to migrate a Modena project to Ultima.

That work was never started, however, we are now in need of upgrading and are looking at using the current Ultima with current primeface version.

Currently we are using Moden 2.1.3 and primeface 6.2.

Has any tools been developed for the conversion? And or, do you have developers that can kick-start our migration? Our developers are not versed in SASS....only CSS ( not sure what that even means)......and are not clear if a complete re-write needs to be done and a start from scratch is in order.

Is there a step-by-step guide how to start a project with Ultima 7 or even with Ultima 1.0? Our developers use Netbeans.  Ideally, I want to keep our current template - just convert or import it and fix code that needs fixing.

Any help or guidance would be much appreciated.
Thanks,
Chuck

**Comment by melloware** [comment, marked as answer]
Unfortunately there is no step by step guide and this conversion will have to be done by hand since you are changing from a legacy premium theme to a newer premium theme.
